Example/ReusableDemoTests/ReusableDemoTests.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ReusableDemoTests.swift
+//  ReusableDemoTests
+//
+//  Created by Ben Chatelain on 7/8/16.
+//  Copyright © 2016 AliSoftware. All rights reserved.
+//
+
+import XCTest
+
+class ReusableDemoTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    }
+}
```

Comment:
Yeah, it's just the auto-generated example tests file. I started on #7 in this effort, but then decided to wait until the API settled before actually writing tests.